streaming/python/runtime/worker.py

```diff
@@ -104,3 +223,164 @@ def on_writer_message_sync(self, buffer: bytes):
             return _NOT_READY_FLAG_
         result = self.writer_client.on_writer_message_sync(buffer)
         return result.to_pybytes()
+
+    def shutdown_without_reconstruction(self):
+        logger.info("Python worker shutdown without reconstruction.")
+        ray.actor.exit_actor()
+
+    def notify_checkpoint_timeout(self, checkpoint_id_bytes):
+        pass
+
+    def commit(self, barrier_bytes):
```

Comment:
bool can be serialzied by ray, why using protobuf. This just compilcated code

Reply:
I'll do this in separate PR.